tsconfig_dev.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
hmm to check my understanding -- what's the separate `_dev.json` for?

tried replacing `npm run dev: "tsc --build --watch"`, seems to work too

Reply:
If I can summarize, `tsconfig.json` is intended for production as it does not want to compile if there are errors as a [consequence](https://www.typescriptlang.org/docs/handbook/project-references.html#caveats) to the [project references](https://www.typescriptlang.org/docs/handbook/project-references.html) approach, and `tsconfig_dev.json` is intended for development as it allows that to happen, facilitating tests in WIP (e.g. testing if the import/export syntax plays well with other files before moving on to fixing the types), albeit with a performance hit.